src/Command/CommandHelper.php

```diff
@@ -179,7 +179,15 @@ public static function begin(
 			})($autoloadFile);
 		}
 		if ($projectConfigFile === null) {
-			foreach (['phpstan.neon', 'phpstan.neon.dist', 'phpstan.dist.neon'] as $discoverableConfigName) {
+			$discoverableConfigNames = [
+				'.phpstan.neon',
+				'.phpstan.neon.dist',
+				'.phpstan.dist.neon',
+				'phpstan.neon',
+				'phpstan.neon.dist',
+				'phpstan.dist.neon',
```

Comment:
Is it wanted to load `.phpstan.neon.dist` rather than `phpstan.neon` or should it be
```
				'.phpstan.neon',
				'phpstan.neon',
				'.phpstan.neon.dist',
				'phpstan.neon.dist',
				'.phpstan.dist.neon',
				'phpstan.dist.neon',
```
?

Reply:
@VincentLanglet Please send an update.